src/ConnectionConfig.test.tsx

```diff
@@ -192,4 +193,27 @@ describe('ConnectionConfig', () => {
     await waitFor(() => expect(screen.getByTestId('connection-config')).toBeInTheDocument());
     expect(screen.queryByText('Assume Role ARN')).toBeInTheDocument();
   });
+  it('should not render externalId field if GrafanaAssumeRole auth type is selected', async () => {
+    (window as any).grafanaBootData = {
+      settings: {
+        awsDatasourcesTempCredentials: true,
+      },
+    };
+    const props = getProps({ options: { jsonData: { authType: AwsAuthType.GrafanaAssumeRole } } });
+    render(<ConnectionConfig {...props} />);
+
+    await waitFor(() => expect(screen.queryByLabelText('External ID')).not.toBeInTheDocument());
+  });
+  it('should not render GrafanaAssumeRole as auth type if the feature flag is not enabled', async () => {
```

Comment:
maybe lets add a test for the positive case as well? (it shows grafana assume role when enabled?